controllers/provider-azure/charts/internal/shoot-storageclasses/templates/storageclasses.yaml

```diff
@@ -21,6 +21,15 @@ parameters:
 ---
 apiVersion: {{ include "storageclassversion" . }}
 kind: StorageClass
+metadata:
+  name: managed-standard-ssd
+provisioner: kubernetes.io/azure-disk
+parameters:
+  storageaccounttype: StandardSSD_LRS
```

Comment:
This storageclass is only available for Kubernetes >= 1.13 - ref https://github.com/kubernetes/kubernetes/issues/73260#issuecomment-457252026.
Can we pass the Shoot Kubernetes version as a value and do not apply this storage class when the version is < 1.13?

```
$ k describe pvc azure-managed-disk
Events:
  Type       Reason              Age               From                         Message
  ----       ------              ----              ----                         -------
  Warning    ProvisioningFailed  6s (x3 over 27s)  persistentvolume-controller  Failed to provision volume with StorageClass "managed-standard-ssd": azureDisk - StandardSSD_LRS is not supported sku/storageaccounttype. Supported values are [Premium_LRS Standard_GRS Standard_LRS Standard_RAGRS Standard_ZRS]
Mounted By:  <none>
```